tests/fixtures/quick_fetch.yml

```diff
@@ -0,0 +1,313 @@
+http_interactions:
+- request:
+    method: get
+    uri: https://api.meetup.com/rladies-nashville/events?status=past&offset=0
+    body:
+      encoding: ''
+      string: ''
+    headers:
+      Accept: application/json, text/xml, application/xml, */*
+      Authorization: Bearer <<<my_access_token>>>
```

Comment:
the secret has been hidden by the vcr_configuration you can see in the tests/testthat/setup.R file.

Reply:
Cf https://github.com/ropensci/vcr/issues/200 I think the config is not entirely secure yet but I did check my cassettes were ok.